installer_scripts/yb_migrate_installer__centos7.sh

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
+
+# This script installs yb_migrate in /usr/local/bin and all of its dependencies.
+
+LOG_FILE=yb_migrate_installer.log
+
+export GOROOT="/tmp/go"
+
+ORA2PG_VERSION="23.1"
+GO_VERSION="go1.18"
+YUM="sudo yum install -y -q"
+GO="$GOROOT/bin/go"
+
+# separate file for bashrc related settings
+RC_FILE="$HOME/.yb_migrate_installer_bashrc"
+touch $RC_FILE
+# just in case last execution of script stopped in between
+source $RC_FILE
+
+
+main() {
+	set -x
+	output "Installing RPM dependencies."
+	$YUM https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm 1>&2
+	$YUM which wget git gcc perl-5.16.3 perl-DBI-1.627 perl-App-cpanminus mysql-devel postgresql14-server 1>&2
+
+	install_golang
+
+	output "Install Oracle instant clients."
+	OIC_URL="https://download.oracle.com/otn_software/linux/instantclient/215000"
+	$YUM \
```

Comment:
Just one concern, in case oracle clients setup exists on user machine, what will happen in that case?
Will YUM take care of that in case when `20.5.0.0-1` version of clients is already there?